cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1846,6 +1846,9 @@ class ClusterSchema(BaseSchema):
     additional_resources = fields.Str(metadata={"update_policy": UpdatePolicy.SUPPORTED})
     dev_settings = fields.Nested(ClusterDevSettingsSchema, metadata={"update_policy": UpdatePolicy.SUPPORTED})
     deployment_settings = fields.Nested(DeploymentSettingsSchema, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
+    disable_sudo_default_user = fields.Bool(
```

Comment:
*Minor*: something we name variable/classes like "disable_sudo_default_user" and other time "disable sudo *access*". I suggest to align the naming to avoid misunderstandings (even if really low probable in this case).